README.md

```diff
@@ -59,22 +59,32 @@ $ ./go init
 $ ./go serve
 ```
 
-You should now be able to visit `http://127.0.0.1:4000/web-design-standards/`
+You should now be able to visit `http://127.0.0.1:4000/`
 and view the web design standards locally.
 
 Questions or need help with setup? Feel free to open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
 
-### Contributing to the code base
+## Contributing to the code base
 
-See [CONTRIBUTING](CONTRIBUTING.md)
+See [CONTRIBUTING](CONTRIBUTING.md).
 
-### Got feedback?
+## Got feedback?
 
-Please create a [GitHub Issue](https://github.com/18F/web-design-standards/issues)
+Please create a [GitHub Issue](https://github.com/18F/web-design-standards/issues).
 
-### Public domain
+## Licenses and attribution
 
-This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
+### A few parts of this project are not in the public domain
+
+The files in `assets/img` are from [Font Awesome](http://fontawesome.io/) by Dave Gandy. They are under the [SIL Open Font License 1.1](http://scripts.sil.org/OFL).
```

Comment:
Source Sans Pro and Merriweather are also released on SIL Open Font License.  @maya @jpyuda do we also need to list those here? (See language here on font licenses: https://github.com/18F/web-design-standards-assets/pull/9/files)